torch/distributed/elastic/multiprocessing/__init__.py

```diff
@@ -88,6 +88,7 @@ def start_processes(
     args: Dict[int, Tuple],
     envs: Dict[int, Dict[str, str]],
     log_dir: str,
+    log_line_headers: Dict[int, str],
```

Comment:
to make programmatic usages BC, `log_line_headers` needs to be made `Optional[Dict[int, str]]` and defaulted to `None`.